check/directory.go

```diff
@@ -9,28 +9,50 @@ import (
 )
 
 const (
+	CdktfIndexDirectory = `cdktf`
+
 	DocumentationGlobPattern = `{docs/index.md,docs/{data-sources,guides,resources},website/docs}/**/*`
 
 	LegacyIndexDirectory       = `website/docs`
-	LegacyDataSourcesDirectory = `website/docs/d`
```

Comment:
These constants are exported and therefore part of the Go module compatibility promises, however in spot checking various dependents of this Go module, it does not appear that any drop down to using the Go code directly so it's likely fine that this is being changed.

Reply:
Oh I see, I wasn't aware :) Thanks for raising this and thank you for checking if it causes trouble down the road :)